core/src/main/scala/io/finch/internal/Accept.scala

```diff
@@ -0,0 +1,22 @@
+package io.finch.internal
+
+import cats.Eq
+import javax.activation.MimeType
+import scala.util.control.NonFatal
+
+case class Accept(primary: String, subtype: String)
```

Comment:
As `NegotiateToResponse` lives under a root namespace, maybe we should move this too (one level up)?